lib/inputstreamhelper.py

```diff
@@ -88,17 +88,7 @@ def install_widevine():
 
 def remove_widevine():
     ''' API entry point for removing Widevine CDM '''
-    # NOTE: Pretty crude way of ensuring CDM is removed
-    cdm_path = xbmc.translatePath(xbmcaddon.Addon('inputstream.adaptive').getSetting('DECRYPTERPATH'))
-    for extension in config.CDM_EXTENSIONS:
-        widevinecdm = os.path.join(cdm_path, 'libwidevinecdm' + extension)
-        if os.path.exists(widevinecdm):
-            log('Remove Widevine CDM at {0}'.format(widevinecdm))
-            os.remove(widevinecdm)
-            xbmcgui.Dialog().ok('{0} v{1}'.format(ADDON_ID, ADDON_VERSION), LANGUAGE(30051))
-            break
-    else:
-        xbmcgui.Dialog().ok('{0} v{1}'.format(ADDON_ID, ADDON_VERSION), LANGUAGE(30052))
+    Helper('mpd', drm='widevine').remove_widevine()
```

Comment:
Maybe you call this directly from addon.py, like I am doing in #98

Reply:
Okay, I rebased and fixed this.